DataManagementSystem/Agent/FTSAgent.py

```diff
@@ -925,7 +925,10 @@ def __finalizeFTSJob( self, request, ftsJob ):
     if not processFiles["OK"]:
       log.error( processFiles["Message"] )
       return processFiles
-    ftsFilesDict = self.updateFTSFileDict( ftsFilesDict, processFiles["Value"] )
+    processFiles = processFiles['Value']
+    if processFiles['toRegister']:
+      log.error( "%f files could not be registered in FC" % len( processFiles['toRegister'] ) )
```

Comment:
If you want to continue the "cleaning" of the FTSAgent, please go ahead, but I gave up, much more important things to do...